javascript-light-bulb/main.js

```diff
@@ -0,0 +1,18 @@
+const $lightbulb = document.querySelector('.bulb');
+const $body = document.querySelector('body');
+const bulbList = $lightbulb.classList;
+
+function changeBulb(event) {
+  console.log('Callback called!');
```

Comment:
Remove all `console.log` before submitting, they are only for testing & debugging purposes!